.github/actions/bump-version/index.js

```diff
@@ -0,0 +1,8 @@
+const action = require("./action");
+const core = require("./core");
+
+action.bumpVersion(
+  core.getInput("currentVersion"),
+  core.getInput("bumpType"),
+  core.getInput("prereleaseSuffix")
+);
```

Comment:
Got rid of `fs.readFileSynx`, `fs.writeFileSynx`, and doing anything with the `newVersion` that used to get returned from the `action.bumpVersion` call. `bumpVersion` uses `core.setOutput` and then we use the output in the `release.yaml` workflow.